src/main.rs

```diff
@@ -1,3 +0,0 @@
-fn main() {
-    println!("Hello, world!");
```

Comment:
You actually can create new project using `--lib` flag, which will not create `main.rs`. Instead `lib.rs` is created.

Reply:
thanks